docs/Using-Fleet/Vulnerability-Processing.md

```diff
@@ -3,6 +3,7 @@
 - [Configuration](#configuration)
 - [Performance](#performance)
 - [Detection pipeline](#detection-pipeline)
+- [What Fleet checks for vulnerabilites](#what-fleet-checks-for-vulnerabilites)
```

Comment:
```suggestion
- [Coverage](#coverage)
```

Since we updated the new section's title to "Coverage," I think we'll need to update this link to "Coverage."

In addition, I think we can move the new "Coverage" section up so that it is the second section on the page (Following "What to expect"). This means, that the coverage link would move below the "What to expect" link.

This is because we expect that the user visiting this page is primarily attempting to achieve the goal of understanding the vulnerability coverage Fleet offers.